test/src/main/java/org/apache/accumulo/test/functional/MemoryStarvedMajCIT.java

```diff
@@ -89,7 +88,8 @@ public static void start() throws Exception {
           }
           if (line.startsWith("accumulo")) {
             Metric metric = TestStatsDSink.parseStatsDMetric(line);
-            if (MINC_PAUSED.getName().equals(metric.getName())) {
+            if (org.apache.accumulo.core.metrics.Metric.MAJC_PAUSED.getName()
```

Comment:
```suggestion
            if (MAJC_PAUSED.getName()
```

Reply:
I had to specify the full path since there was another variable with the same name in scope. I renamed the other variable to make things less confusing and to avoid having use the full path.

Reply:
I see. Change looks good